src/code.rs

```diff
@@ -441,6 +442,51 @@ impl {struct_name} {{
     }}
 "##));
 
+    // Table::filter() helper fn
+    {
```

Comment:
this block should likely be its own function to keep the functions smaller, but this can be delayed until we refactor this (see #105)

Reply:
Awesome, I’ll take a look at this later and try to rebase 🙌

Reply:
unless you want to delay this PR, i will hold off on merging other stuff (even if approved) so you can get this (and possibly your other PR's) merged without having to rebase it often

Reply:
Thanks @hasezoey, I really appreciate your consideration 🤗 

Please do approve this PR if you think it's generally acceptable. That way, I can rebase and merge later without having to stop any others.

Reply:
> Please do approve this PR if you think it's generally acceptable. That way, I can rebase and merge later without having to stop any others.

i take this as meaning you have not rebased yet, so i will merge the other standing (approved) PRs to not have to wait, after that i will approve